doc/memos/rdm-draft-weiss-exposing-api-firmware-guidelines.md

```diff
@@ -0,0 +1,155 @@
+- rdm-draft-weiss-exposing-api-firmware-guidelines.md
+- Title: Guidelines for Write Firmware to Expose RIOT APIs
+- Author: Kevin Weiss
+- Status: In-progress
+- Type: Process
+- Date: December 2018
+
+## Abstract
+
+In order to improve the testing and interface tools for RIOT a standard way of
+exposing API calls to a higher level should be defined. This document describes
```

Comment:
YES: It will be really helpful to make the context and subject of consideration crisply clear so that the reader understands right away what to expect from this memo.

Reply:
Ditto, NB that "exposing API calls" refers to a module exposing functions for other modules to use. Not to a human

Reply:
OK, I will try to rephrase it in a clearer way.